web/src/components/modals/ErrorModal.jsx

```diff
@@ -26,15 +26,15 @@ export default function ErrorModal(props) {
       className="Modal DefaultSize"
     >
       <div className="Modal-body">
-        <div className="flex justifyContent--flexEnd">
+        <div className="tw-flex tw-justify-end">
           <Icon
             icon="close"
             size={14}
             className="gray-color clickable close-icon"
             onClick={() => toggleErrorModal()}
           />
         </div>
-        <div className="flex flex-column justifyContent--center alignItems--center">
+        <div className="tw-flex tw-flex-col tw-justify-center tw-items-center">
```

Comment:
This looks fine: 

## CURRENT
<img width="1159" alt="Screenshot 2022-12-19 at 10 39 14" src="https://user-images.githubusercontent.com/4998130/208475932-2fa3e2a4-9044-407a-a70d-d35bc48267a5.png">



### NEW
<img width="1159" alt="Screenshot 2022-12-19 at 10 41 25" src="https://user-images.githubusercontent.com/4998130/208475900-a190e58a-9df5-4978-9daf-5c506112badb.png">